src/analyses/dependence_graph.cpp

```diff
@@ -27,19 +27,20 @@ bool dep_graph_domaint::merge(
   goto_programt::const_targett from,
```

Comment:
Nit: this commit message (Fix issue where merge() did not return true in the dependence graph w) isn't rendering correctly so I suggest the title is too long. Suggest making the short text:

```
Make dependency graph merge() return true when abstract state changes
```

I think that is short enough